docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+preferences. In most cases, you can use the DD Mon YYYY or DD Month YYYY format for dates. 
+In body text, spell out whole numbers from zero through nine. Use numerals for 10 or greater. 
+Use commas in numbers with four or more digits. Use *more than* instead of *over* 
+(over is a spatial term). 
+
+### Emphasis and special notices
+
+Use bold formatting for user interface elements that the user interacts with, including:
+
+- Menus and menu commands.
+- Field and button labels.
+- Options that can be selected in the user interface.
+
+If there's text that requires more attention than the surrounding body, consider isolating 
+it as a standalone note or tip. However, **think twice** before adding any type of special 
+notice, embedding details, or using tabbed boxes. These elements can all be distracting and 
+too many of them make text unreadable. 
+
+Because they are generally **disruptive** to the reader's experience, ask yourself if it is 
+really necessary to stop the reader's forward progress by adding a Note, Caution, Tip, or 
+any other special component. In most cases, it's more effective to integrate the information 
+in the main body of the text. If you use these types of components at all, use them sparingly! 
+
+**Note**—Indicates neutral or positive information that emphasizes or supplements important 
+points of the main text. A note supplies information that may apply only in special cases. 
+Examples are memory limitations, equipment configurations, or details that apply to specific 
+versions of a program.
+
+**Tip**—Helps users apply the techniques and procedures described in the text to their specific 
+needs. A tip suggests alternative methods that may not be obvious and helps users understand 
+the benefits and capabilities of the product. A tip is not essential to the basic understanding 
+of the text.
+
+**Caution**—Advises users that failure to take or avoid a specific action could result in 
+loss of data.
+
+### Images
+
+Diagrams and illustrations can help readers visualize and internalize complex ideas and 
+processes. So, use them liberally but with intention. Images also help to break up long text 
+flows, but they should always reinforce and reflect the text immediately preceding or 
+immediately following the image.
+
+If you include screenshots, only include the relevant parts of the screen and use callouts 
+to highlight how what is captured in the image is relevant to the text.
+
+Be wary of using diagrams or illustrations that include any information—visual or textual—that 
+is likely to get stale.
+
+### Log in and sign in formats
+
+Most Linux distributions and macOS use *log in* to describe how a user initiates an interactive 
+session. Windows uses *log on*. Many modern web-based applications—including the Teleport Web 
+UI—use *sign in*.
+
+- Use sign in or log in as two words with no hyphen when describing an action (verb usage).
```

Comment:
```suggestion
- Use "sign in" or "log in" as two words with no hyphen when describing an action (verb usage).
```
Or italicize these terms. Otherwise, this is harder to read.